UpdateSvcutilCorePackage.ps1

```diff
@@ -0,0 +1,122 @@
+$logfile = Join-Path $PSScriptRoot 'UpdateSvcutilCorePackage.log'
```

Comment:
Can we move this script out of the root?
We currently put scripts used for setup and other purposes under...
wcf\src\System.Private.ServiceModel\tools\scripts
It could go there or if you prefer another similar tools\scripts under svcutilcore.